src/scripting/lua_kernel_base.cpp

```diff
@@ -259,7 +259,6 @@ lua_kernel_base::lua_kernel_base(CVideo * video)
 		{ "coroutine",   luaopen_coroutine   },
 		{ "debug",  luaopen_debug  },
 		{ "os",     luaopen_os     },
-		{ "bit32",  luaopen_bit32  }, // added in Lua 5.2
```

Comment:
This should probably be kept around a bit longer, though I don't imagine much UMC actually uses it.